src/parameterStore.ts

```diff
@@ -19,23 +19,56 @@ function getSSM() {
     return new AWS.SSM();
 }
 
-// eslint-disable-next-line import/prefer-default-export
-export async function getIntegrationTransformUrl(): Promise<string> {
-    const ssm = getSSM();
-    let path = process.env.INTEGRATION_TRANSFORM_PATH;
+function getParamPath(paramEnvVarPath: string, defaultPath: string): string {
+    const path = process.env[paramEnvVarPath];
     if (path === undefined) {
-        throw new Error('INTEGRATION_TRANSFORM_PATH is not defined in environment variables');
+        throw new Error(`${paramEnvVarPath} is not defined in environment variables`);
     }
     if (IS_OFFLINE === 'true') {
-        path = `fhir-service.integration-transform.${AWS_REGION}.${STAGE}.url`;
+        return defaultPath;
     }
 
+    return path;
+}
+
+async function getParams(paramStorePaths: string[]): Promise<Record<string, string>> {
+    const ssm = getSSM();
+
     const data = await ssm
-        .getParameter({
-            Name: path,
+        .getParameters({
+            Names: paramStorePaths,
             WithDecryption: true,
         })
         .promise();
 
-    return data.Parameter!.Value!;
+    if (data.InvalidParameters === undefined || data.InvalidParameters.length > 0) {
```

Comment:
Why does `data.InvalidParameters === undefined` causes an error?

Reply:
`eslint` tagged that `InvalidParameters` could be undefined and this was an attempt to fix the `eslint` issue. But actually using `!` is a better way to handle it. 

I'm going to change it to `if (data.InvalidParameters!.length > 0) {`

Reply:
That'd be better. Although in that case you need to make sure that the response always includes `InvalidParameters`(even if it is an empty array)

btw Typescript is the one that highlights this error. eslint doesn't care

Reply:
Looking at the [docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SSM.html#getParameters-property). It looks like `InvalidParameters` is always present, but is an empty array when there are no `invalid` parameters.

Reply:
Sounds good. I think I've seen similar situations with other aws response types. it seems that the Type definitions for the aws-sdk are not that great...